test/index.spec.js

```diff
@@ -284,6 +282,9 @@ describe('jigawatt/index.js', () => {
       const test2 = () => JW(m1, [m2, m3]);
       expect(test2).to.throw(Error);
 
+      const test3 = () => JW({}, [m2, [m1, m3]]);
+      expect(test3).to.throw(Error);
```

Comment:
I think your test should be a little more robust.  What if some sort of generic error is thrown by something other than PropsCheck?